Source/WTF/wtf/MathExtras.h

```diff
@@ -444,6 +444,18 @@ inline constexpr unsigned fastLog2(uint64_t value)
     return fastLog2(static_cast<unsigned>(value));
 }
 
+template <typename T>
+inline constexpr bool isAligned(T* ptr, int alignment = sizeof(void*))
+{
+    return reinterpret_cast<uintptr_t>(ptr) % alignment == 0;
+}
+
+template <typename T, int alignment = sizeof(void*)>
+inline constexpr bool isAligned(T* ptr)
+{
+    return isAligned(ptr, alignment);
+}
```

Comment:
This doesn’t seem like the right header for this. Pointer things don’t really fall into the "Math" category. There are other suitable headers, like `StdLibExtras.h`. Maybe you can find an even better one.